eduvpn/brand.py

```diff
@@ -1,20 +1,22 @@
 from eduvpn.util import get_prefix
+from typing import Tuple
 
 prefix = get_prefix()
 
-eduvpn_main_logo = prefix + "/share/icons/hicolor/128x128/apps/eduvpn-client.png"
-eduvpn_name = "eduVPN"
-lets_connect_main_logo = prefix + "/share/icons/hicolor/128x128/apps/lets-connect-client.png"
-lets_connect_name = "Let's Connect!"
+eduvpn_main_logo = prefix + "/share/icons/hicolor/" \
+                            "128x128/apps/eduvpn-client.png"  # type: str
+eduvpn_name = "eduVPN"  # type: str
+lets_connect_main_logo = prefix + "/share/icons/hicolor/128x128" \
+                                  "/apps/lets-connect-client.png"  # type: str
+lets_connect_name = "Let's Connect!"  # type: str
 
 
-def get_brand(lets_connect):
+def get_brand(lets_connect):  # type: (bool) -> Tuple[str, str]
     """
-    args:
-        lets_connect (bool): Let's connect mode?
-
-    returns:
-        (str, str): logo, name
+    args:	
```

Comment:
ik snap deze niet, er is een diff maar de code lijkt het zelfde?

Reply:
Hier is de code twee regels opgeschoven.